database/db.go

```diff
@@ -217,15 +224,21 @@ func NewDbFromConfig(c *Config, logger *logging.Logger, connectorCallbacks Retry
 		DB:              db,
 		Options:         &c.Options,
 		columnMap:       NewColumnMap(db.Mapper),
-		addr:            addr,
+		addrDescription: addrDescription,
 		logger:          logger,
 		tableSemaphores: make(map[string]*semaphore.Weighted),
 	}, nil
 }
 
-// GetAddr returns the database host:port or Unix socket address.
+// GetAddr returns the Redis connection address in a technical form.
 func (db *DB) GetAddr() string {
-	return db.addr
+	return db.addrDescription
+}
+
+// MarshalLogObject implements zapcore.ObjectMarshaler, adding the database address to each log message.
+func (db *DB) MarshalLogObject(encoder zapcore.ObjectEncoder) error {
+	encoder.AddString("database_address", db.addrDescription)
+	return nil
```

Comment:
In general, please add a newline before return as it improves readability.

I know that there is no specific convention or rule in Go regarding the use of newlines before return statements. Even the official Go style guide does not explicitly mention anything in this regard. But we do this in PHP as well and the majority of the code in Icinga DB and Icinga Go library already does this, so I would make this a convention for us.

Reply:
Does this really improves readability for a two line function? As long as it's unique all over the code, I am fine.